internal/distro/rhel86/package_sets.go

```diff
@@ -661,6 +661,7 @@ func anacondaPackageSet(t *imageType) rpmmd.PackageSet {
 			"librsvg2",
 			"linux-firmware",
 			"lklug-fonts",
+			"lldpad",
```

Comment:
Just out of curiosity: Is this only required for dracut on arm64 or was the whole image-installer broken on 8.6 until now?

Reply:
It's required by both.

Had to look into this to make sure if it needs to be included.  I added it because it was mentioned in the failing dracut stage, but it's actually pulled in by `fcoe-utils`, which was already there on 8.6 and I had to add to 9.0.

I can remove it from both since it's a dependency.